controller/api/destination/watcher/endpoints_watcher_test.go

```diff
@@ -2046,3 +2051,133 @@ status:
 		})
 	}
 }
+
+// Test that when an EndpointSlice is scaled down, the EndpointsWatcher sends
```

Comment:
nit: godoc should start with the function name

Reply:
I don't think we typically use godoc for tests, do we? (note the // rather than ///)

Reply:
Ok never mind, I thought one could use `go doc` to show docs from test functions, but that doesn't appear to be the case. OTOH, I'm not aware of the usage of `///`, can you clarify?

Reply:
Nevermind, `///` isn't a go thing.  I got confused; holiday brain.